.github/workflows/cut_release.yml

```diff
@@ -13,11 +13,11 @@ jobs:
           sudo apt-get install -y libxml2-utils python3
           pip3 install lxml beautifulsoup4
 
-      - name: Set up JDK 11
+      - name: Set up JDK 17
         uses: actions/setup-java@v4
         with:
           distribution: 'corretto'
-          java-version: '11'
```

Comment:
Are we able to support multiple versions using the matrix strategy? https://docs.github.com/en/actions/how-tos/write-workflows/choose-what-workflows-do/run-job-variations

I'd also want to see how this would impact our release process if we were producing multiple JDK version releases. cc: @AbdulR3hman @ejeffrli @aimethed